libdl/VITABUILD

```diff
@@ -0,0 +1,18 @@
+pkgname=libdl
```

Comment:
Can you change the pkgname to vita-libdl and adjust all references to vita-libdl later to refer pkgname instead? (https://github.com/vitasdk/packages/blob/master/vitaShaRK/VITABUILD Take this as example).

Reply:
Done